uploader/s3.go

```diff
@@ -38,20 +38,23 @@ type S3Backend struct {
 }
 
 // NewS3Backend creates a new instance of an S3 manager which uploads to the specified location.
-func NewS3Backend(m metrics.Reporter, bucket, prefix, iamRoleArn, taskID string, useDefaultRole bool) (Backend, error) {
-	log.StandardLogger().Infof("logging to %s/%s using iamRole %s", bucket, prefix, iamRoleArn)
-
+func NewS3Backend(m metrics.Reporter, bucket, prefix, taskRole, taskID, writerRole string, useDefaultRole bool) (Backend, error) {
```

Comment:
We keep `useDefaultRole` for backward compatibility? As in the future control plane should always set writeRole.

Reply:
yes. when we're confirmed that the future control plane is always setting `s3WriterRole` we can remove the `useDefaultRole` logic